features/demo/src/manifest.json

```diff
@@ -4,6 +4,8 @@
     "description": "Demonstrates some polyPod capabilities for testing purposes - please ignore",
     "primaryColor": "#000000",
     "thumbnail": "images/thumbnail.pdf",
+    "borderSize": "1",
+    "borderColor": "#cbd5e0",
```

Comment:
I got the impression that border color should be derived from built/preview state. How this borderColor will affect that?